### ISSUE TITLE
[Enhancement] Create VM Disk from QCOW2 File

### ISSUE THREAD

**Issue by ZaxLofful** [Telmate/terraform-provider-proxmox#854]
Cloning a VM from a template is ok....

Making it so you can create a VM from a cloud image already in QCOW2 format, would be the most ideal.

**Comment by myplixa** [comment]
Good afternoon.
I agree, this functionality is very lacking when working with this provider.
There is an "import-from" parameter in the API and I use it with Ansible when deploying a new vm from an official disk image.
I would also like to use not only cloning of ready-made template, but also work with official disk images, it will definitely expand the functionality and add popularity to the Proxmox platform.
One person has already written a similar functionality, but it looks like the development is no longer supported.
https://github.com/danitso/terraform-provider-proxmox

**Comment by ZaxLofful** [comment]
Yeah, setting it up with a few defaults, that use official cloud-init images, like Ubuntu and Red Hat of various versions would be awesome. Then you could also add your own, when the need arises.

**Comment by Dune4** [comment]
I am also looking for this functionality.
